examples_tests/17.SimpleBulletIntegration/CMakeLists.txt

```diff
@@ -4,4 +4,12 @@ if(NOT RES)
 	message(FATAL_ERROR "common.cmake not found. Should be in {repo_root}/cmake directory")
 endif()
 
-irr_create_executable_project("" "")
\ No newline at end of file
+set(BULLET_COLLISION_SRC "" CACHE STRING "For example 17.SimpleBulletIntegration - Not required")
```

Comment:
what happens if can't find Bullet on system? Can it build without?

(BTW there is some code in the top-level CMake that skips building with Vulkan if Vulkan SDK is not detected on the system, maybe you can steal that approach)

Reply:
I'd assume it won't spew errors and you'd be able to do without but just to be safe I'll add some sanity checks if they leave it blank and not glob_recurse over a blank string

Reply:
now your Bullet lib needs to come from the submodule